tools-local/Microsoft.ML.AutoML.SourceGenerator/Microsoft.ML.AutoML.SourceGenerator.csproj

```diff
@@ -0,0 +1,43 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <LangVersion>9.0</LangVersion>
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+    <GetTargetPathDependsOn>$(GetTargetPathDependsOn);GetDependencyTargetPaths</GetTargetPathDependsOn>
+    <IncludeBuildOutput>false</IncludeBuildOutput>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="3.9.0" PrivateAssets="all" />
+    <PackageReference Include="Microsoft.CodeAnalysis.Analyzers" Version="3.3.2" PrivateAssets="all" />
+    <PackageReference Include="Newtonsoft.Json" Version="12.0.2" PrivateAssets="All" GeneratePathProperty="true" />
```

Comment:
Do  you need Newtonsoft or can you use System.Text.Json?

Reply:
The search space generator uses `JToken` to iterate over json so if I use `System.Text.Json` that code probably needs to be re-implemented.

Since it's in local-tool folder and won't pollute released package, maybe we can still use Newtonsoft?

Reply:
The only thing I was thinking of is that @eerhardt said we have a long standing issue to remove it from the code. So I think it would be better to remove it. How much work would the re-write be?

Reply:
Let's stop making new code use Newtonsoft.Json.

Reply:
OK,,, re-written with System.Text.Json

### Resolved